docs/tutorials/maps.ipynb

```diff
@@ -10,15 +10,16 @@
     "\n",
```

Comment:
<p>One could show the newly introduced <code>.interp_to_geom()</code>  as well...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='99'/>